pkg/kritis/metadata/grafeas/grafeas.go

```diff
@@ -232,6 +232,21 @@ func (c Client) CreateAttestationOccurence(note *grafeas.Note,
 	return c.client.CreateOccurrence(c.ctx, req)
 }
 
+// Builds gets Build Occurrences for a specified image.
+func (c Client) Builds(containerImage string) ([]metadata.Build, error) {
+	occs, err := c.fetchOccurrence(containerImage, "BUILD")
```

Comment:
NIT: Add `glog.Infof("getttig build occurrences for %s", containerImage)` log same as containeranalysis one to keep consistency in logs.  (Although, we will not be using this feature)

Reply:
Fixed in 8499f80

Reply:
Fixed in 57a873f